packages/docs/docs/getting-started/time-events.mdx

```diff
@@ -48,3 +48,36 @@ it. This is useful when you want to extend or shorten a pause in your voiceover
 because correcting the first time event after the pause will also fix all events
 after it. You can hold `SHIFT` when editing an event to prevent this from
 happening.
+
+## Controlling animation duration with Time Events
+
+Time Events not only hold information *when* something should occur but also for
+*how long* someting occurs. With this in mind the system also allows to control
+the duration of an animation. Lets say we want to animate a circle for the duration
+of an event.
+First we would need to register a Time Event on the Scene:
+
+```ts
+useScene().timeEvents.register('event')
+```
+
+This registers a Time Event called `event`.
+Then we can grab the duration of said event as follows:
+
+```ts
+const duration = useScene().timeEvents.get('event').offset;
+```
+
+And finally we can use this `duration` to animate the circle:
+
+```ts
+yield* circle().scale(2, duration);
+```
```

Comment:
This is an internal implementation detail, I don't think we should include it in the docs.
I'd simply say:
````markdown
Aside from specifying _when_ something should happen, time events can be used to 
control _how long_ something should last. You can use the 
[`useDuration`](/api/core/utils#useDuration) function to retrieve the duration of an event
and use it in your animation:

```ts
yield* circle().scale(2, useDuration('event'));
```
````